src/utilities/values-2d.ts

```diff
@@ -0,0 +1,16 @@
+import { values } from './values';
+
+/**
+ * Returns a matrix of given width and height with each element filled with the same value
+ */
+export function values2D(
+  width: number,
+  height: number,
+  value: number
+): Float32Array[] {
+  const result = new Array(height);
```

Comment:
I think `Float32Array` constructor should be preffered here.

Reply:
I dont that will work because float32Arrays need to have Float32s inside, not other Float32Arrays.

Reply:
return type is `Float32Array[]` and we are returning `result` here which is `new Array(height)`

Reply:
Ohk. I'll make it result: `Float32Array[]` then.

Reply:
Done ig.